packages/plugin-css/src/index.ts

```diff
@@ -227,7 +227,7 @@ export function transformTransition(value: ParsedTransitionToken['$value']): str
 export function transformTypography(value: ParsedTypographyToken['$value']): Record<string, string | number | string[]> {
   const values: Record<string, string | number | string[]> = {};
   for (const [k, v] of Object.entries(value)) {
-    values[kebabinate(k)] = Array.isArray(v) ? formatFontNames(v) : (v as ParsedTypographyToken['$value']);
```

Comment:
Not sure what type should be used here, so just bypassed type checks for now.

Reply:
I believe that was the correct type?

Reply:
Oh I see the error you’re getting with that type change. Yes, this is fine.

This is some weirdness caused by the translation between all token types with different schemas vs a single token type with a fixed value. Normally TypeScript can discriminate based on `$type`, but when going back the other direction things get messy.

This is fine for now; I’ll clean this up later.